src/server/certificates/controller.test.js

```diff
@@ -94,6 +95,125 @@ describe('#certificatesController', () => {
   })
 })
 
+describe('API calls for GET/PUT/DELETE', () => {
+  /** @type {Server} */
+  let server
+  let mockGetCertificate, mockPutCertificate, mockDeleteCertificate
+  const apiHeaderKey = config.get('apiAuth')
+  const result = {
+    certNumber: 'GBR-2024-CC-123A4BC56',
+    timestamp: '12 MAY 2024',
+    status: 'COMPLETE',
+    isValid: true
+  }
+
+  beforeAll(async () => {
+    server = await createServer()
+    await server.initialize()
+  })
+
+  beforeEach(() => {
+    mockGetCertificate = jest.spyOn(Certificates, 'getList')
+    mockPutCertificate = jest.spyOn(Certificates, 'uploadCertificateDetails')
+    mockDeleteCertificate = jest.spyOn(Certificates, 'removeDocument')
+    mockGetCertificate.mockResolvedValue(result)
+    // mockPutCertificate.mockResolvedValue(result)
```

Comment:
please remove these if they aren't being used

Reply:
Thanks, @alexwbbr - I missed removing the commented code. It was now updated.